code/drasil-website/lib/Drasil/Website/Body.hs

```diff
@@ -89,8 +100,12 @@ sections fl = [headerSec, introSec, gettingStartedSec quickStartWiki newWorkspac
 
 -- | Needed for references and terms to work.
 symbMap :: FolderLocation -> ChunkDB
-symbMap fl = cdb ([] :: [QuantityDict]) (map nw [webName, web] ++ map getSysName allExampleSI)
-  ([] :: [ConceptChunk]) ([] :: [UnitDefn]) [] [] [] [] [] [] [] $ allRefs fl
+symbMap fl = cdb ([] :: [QuantityDict]) (map nw [webName, web, phsChgMtrl] ++ 
```

Comment:
?????

Reply:
The reason I put `phsChgMtrl` there is because it is of the type `CI` whereas the following list that contains terms such as `pendulum`, `motion`, etc. are of type `ConceptChunk`. I tried putting phsChgMtrl with the others but it did not allow me to compile it due to type mismatch